src/wdmp_internal.c

```diff
@@ -840,6 +839,11 @@ void wdmp_form_table_response(res_struct *resObj, cJSON *response)
 			*statusCode = WDMP_STATUS_ATOMIC_GET_SET_FAILED;
 			break;
 		}
+		else if (ret[i] == WDMP_ERR_SESSION_IN_PROGRESS)
+		{
+			*statusCode = WDMP_STATUS_POKE_PROGRESS;
```

Comment:
We cannot mention poke word in webpa, as it is webconfig specific feature.
Add some generic macro "WDMP_STATUS_PREVIOUS_REQUEST_INPROGRESS" etc